### PR TITLE
Updated examples spring 2021

### PR DESCRIPTION
**Pull request by sknick-iastate**
This PR updates the examples to include all proposed changes for spring 2021.

The examples were expanded to include additional road events modeled after the scenarios [here](https://github.com/usdot-jpo-ode/jpo-wzdx/discussions/131).  Below is a summary of the examples that were created:

  - Scenario 1  - Simple Workzone
    - First work zone shown with basic lane information
    - Second work zone shown with detailed lane information
    - Third work zone showing a work zone broken into multiple road events
  - Scenario 2 - Lane Shift (simple case)
    - Single work zone with lane shift
  - Scenario 3 - Shoulder Closure
    - Single work zone but two road events (one in direction of shoulder closure and second in opposite direction)
  - Scenario 4 - Road Closure with Detour on Arterial
    - Single work zone with a detour
  - Scenario 5 - Work Zone with Bridge Work Overhead
    - This is a simple work zone but added a recurring component.  Four additional events were created with a pending status
  - Local Access Example
    - Single work zone with local access created.  Also included bike lane in example for reference.

Throughout the examples, various optional fields were used as well as a variety of enumerations including the verified location/time.  I also tried to remove all depreciated fields. I expect that addition examples can be added in the future based on feedback.

I included reference to the new examples but didn't refer to the scenarios in the [discussions](https://github.com/usdot-jpo-ode/jpo-wzdx/discussions/131).  I think it would be valuable to add the link if this is acceptable.

### CODE REVIEW

**Comment by j-d-b** [comment]
@sknick-iastate wow! Lots of new examples—impressive content.

**Review by j-d-b — CHANGES_REQUESTED** [review]
@sknick-iastate I checked a few of these example files again the schema, using https://www.jsonschemavalidator.net/s/ykYbPPxM, and they did not pass, mostly due to the formatting of `road_names`. Can you validate each example against the current schema (`wzdx_vNext_feed.json` in the `spring-2021-release` branch) and make the necessary changes so that each pass?

**Comment by sknick-iastate** [comment]
> @sknick-iastate I checked a few of these example files again the schema, using https://www.jsonschemavalidator.net/s/ykYbPPxM, and they did not pass, mostly due to the formatting of `road_names`. Can you validate each example against the current schema (`wzdx_vNext_feed.json` in the `spring-2021-release` branch) and make the necessary changes so that each pass?

Thanks @j-d-b !  I went through and resolved the issues.

**Review by chuehlien — COMMENTED** [review]
Very nice and comprehensive examples! Excellent work! The only thing I noticed was that the optional bbox fields at the WZDx feed level and at the RoadEventFeature level were not included in the two comprehensive geojson files. Otherwise, this is great and I can confirm that all examples are passing the JSON schema validator. 

Remember to do a `git merge` with the current `usdot-jpo-ode:spring-2021-release` branch as well.

**Review by j-d-b — APPROVED** [review]

File: create-feed/examples/linestring-examples/scenario1_simple_linestring_example.geojson [hunk 1]
```diff
@@ -0,0 +1,788 @@
...
+         }
+      },
+      {
+         "id": "65773-1",
+         "type": "Feature",
+         "properties": {
+            "data_source_id": "2",
+            "event_type": "work-zone",
+            "relationship": {
+               "first": [
```
[the remark this answers is not in the document]
Comment (sknick-iastate) [reply create-feed/examples/linestring-examples/scenario1_simple_linestring_example.geojson:413]: @j-d-b Thanks!  Everything looks good to me, I just made a few minor formatting changes as well as fixing an error I had.  I think the way you have the relationships is good and utilizing the parent to tie things together.

**Pull request merged**